### PR TITLE
Add linting test and make recommended updates

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,13 @@
 {
-    "dependencies": {
-        "yuglify": "^0.1.4"
-    }
+  "scripts": {
+    "test": "eslint dashboard/static/js/*.js && csslint --quiet --warnings=' ' dashboard/static/css/*.css push/static/css/*.css",
```
Comment (openjck) [inline package.json]: We'll have to be careful to keep these file paths up-to-date as we add more CSS and JS.

File: package.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
 {
-    "dependencies": {
-        "yuglify": "^0.1.4"
-    }
+  "scripts": {
+    "test": "eslint dashboard/static/js/*.js && csslint --quiet --warnings=' ' dashboard/static/css/*.css push/static/css/*.css"
```
Comment (openjck) [inline package.json]: We'll have to be careful to keep these file paths up-to-date as we add more CSS and JS.

File: package.json [hunk 3]
```diff
@@ -1,5 +1,12 @@
 {
-    "dependencies": {
-        "yuglify": "^0.1.4"
-    }
+  "scripts": {
+    "test": "eslint dashboard/static/js/*.js; csslint --quiet --warnings=' ' dashboard/static/css/*.css push/static/css/*.css"
```
Comment (openjck) [inline package.json]: We'll have to be careful to keep these file paths up-to-date as we add more CSS and JS.

Comment (groovecoder) [inline package.json]: Is there a way to make it run against all `*.js` and `.css` files recursively in the current directory? (Except the `node_modules/` directory.)

Comment (openjck) [inline package.json]: Yep, that would be similar to what we did in the Gulp script:

https://github.com/openjck/push-dev-dashboard/blob/jshint/gulpfile.js#L9

I'll make the update

**Comment by openjck** [comment]
Updated

File: package.json [hunk 4]
```diff
@@ -1,5 +1,12 @@
 {
-    "dependencies": {
-        "yuglify": "^0.1.4"
-    }
+  "scripts": {
+    "test": "eslint --ignore-pattern 'staticfiles/**/*.js' --ignore-pattern 'env/**/*.js' .; csslint --quiet --exclude-list='node_modules,staticfiles,env' --warnings=' ' ."
```
Comment (groovecoder) [inline package.json:3]: `staticfiles/` is the output directory of `collectstatic` isn't it? We should `eslint` the `static/` files instead.

**Comment by groovecoder** [comment]
I also updated this branch with the `tox.ini` update and related fixes to run it during Travis.

File: package.json (hunk 4 above)
Comment (openjck) [inline package.json:3]: Yep. The `staticfiles` directory is being ignored.

Comment (groovecoder) [inline package.json:3]: Oh, duh; I over-looked that was an ignore pattern.

**Comment by groovecoder** [comment]
Weird that Travis isn't running the front-style job on this PR commit ... but maybe it will run on master.

**Pull request merged**